src/backend/distributed/sql/downgrades/citus--10.2-5--10.2-4.sql

```diff
@@ -0,0 +1 @@
+#include "../udfs/citus_finish_pg_upgrade/10.2-4.sql"
```

Comment:
We already have the same file in the parent directory. I think we should remove this one.